hadoop-yarn-project/hadoop-yarn/hadoop-yarn-api/src/main/java/org/apache/hadoop/yarn/conf/YarnConfiguration.java

```diff
@@ -4031,6 +4031,11 @@ public static boolean isAclEnabled(Configuration conf) {
   // 5 minutes
   public static final int DEFAULT_FEDERATION_CACHE_TIME_TO_LIVE_SECS = 5 * 60;
 
+  public static final String FEDERATION_CACHE_ENTITY_NUMS =
+      FEDERATION_PREFIX + "cache-entity.nums";
+  // default 1000
```

Comment:
NIT - This comment can be removed as the variable name is appropriate.

Reply:
Thanks for reviewing the code! I will improve the code.